common/src/main/java/com/gentics/mesh/metric/MetricsService.java

```diff
@@ -2,12 +2,12 @@
 
 import java.io.IOException;
 import java.util.Set;
+import java.util.concurrent.atomic.AtomicLong;
 
-import com.codahale.metrics.Counter;
-import com.codahale.metrics.Meter;
-import com.codahale.metrics.MetricRegistry;
-import com.codahale.metrics.Timer;
-
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.DistributionSummary;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Timer;
 import io.vertx.core.buffer.Buffer;
 
 public interface MetricsService {
```

Comment:
I think the interface is ok here, because it is not specific to the server that will scrape the metrics. The current implementation is prometheus specific.